server/command.go

```diff
@@ -16,6 +16,7 @@ const helpText = "###### Mattermost Jira Plugin - Slash Command Help\n" +
 	"\nFor system administrators:\n" +
 	"* `/jira install cloud <URL>` - connect Mattermost to a cloud Jira instance located at <URL>\n" +
 	"* `/jira install server <URL>` - connect Mattermost to a server Jira instance located at <URL>\n" +
+	"* `/jira assign <issuekey> <assignee>` - Changes the assignee of a Jira issue.\n" +
```

Comment:
```suggestion
	"* `/jira assign <issuekey> <assignee>` - Change the assignee of a Jira issue.\n" +
```

Reply:
Changes > Change - for consistency with other commands (e.g. `connect`/`disconnect`)

Reply:
Got it.  Thanks J

Reply:
@jasonblais Do you want me to do the same for /jira transistion while I'm in this area of the code?

Reply:
@jfrerich go for it 👍